ui/raidboss/data/07-dt/raid/r2s.ts

```diff
@@ -101,9 +106,28 @@ const triggerSet: TriggerSet<Data> = {
     {
       id: 'R2S Headmarker Spread',
       type: 'HeadMarker',
-      netRegex: { id: headMarkerData.spreadMarker2, capture: false },
-      suppressSeconds: 5,
-      response: Responses.spread(),
+      netRegex: { id: headMarkerData.spreadMarker2, capture: true },
+      delaySeconds: (data, matches) => {
+        data.beatTwoSpreadCollect.push(matches.target);
+        return data.beatTwoSpreadCollect.length === 2 ? 0 : 0.1;
+      },
+      alertText: (data, _matches, output) => {
+        if (data.beatTwoSpreadCollect.length === 0)
+          return;
+
+        if (data.beatTwoSpreadCollect.includes(data.me))
+          return output.spread!();
+        else if (data.beatTwoOneStart)
+          return output.towers!();
+      },
+      run: (data) => {
+        data.beatTwoSpreadCollect = [];
+        data.beatTwoOneStart = false;
+      },
+      outputStrings: {
+        spread: Outputs.spread,
```

Comment:
Given the whole substance of this PR, is it maybe worth doing a custom response along the lines of `Spread -- Avoid Towers`?

Reply:
Yea we should probably make it a response. I'll look into that.